src/main/java/com/gmail/goosius/siegewar/settings/SiegeWarSettings.java

```diff
@@ -763,19 +766,42 @@ private static void addSpecialEffects(ItemStack artefact, ItemMeta itemMeta, Str
         }
 		//Add special effects
 		for(String[] effectSpec: effectSpecs) {
+		    if (!validateEffect(material, effectSpec[0]))
+		        return false;

```

Comment:
🚧 
If I am reading this correctly, this method would return false for a custom effect, thus (presumably unintendely) leading to a severe error on startup.

Reply:
Woops, that was something I switched around because it didn't look right after I coded it, but you've caught it. Will remove the ! from the custom_effect test.